apps/crn-server/src/controllers/research-outputs.ts

```diff
@@ -79,46 +79,51 @@ export default class ResearchOutputs implements ResearchOutputController {
   async create(
     researchOutputCreateData: ResearchOutputCreateData,
   ): Promise<ResearchOutputResponse | null> {
-    await this.validateResearchOutput(researchOutputCreateData);
+    const normalizedResearchOutputCreateData = this.normaliseResearchOutputData(
```

Comment:
you mixed BE and AE spelling here (normalize vs normalise), I would recommend using BE 
```suggestion
    const normalisedResearchOutputCreateData = this.normaliseResearchOutputData(
```